dropwizard-auth/src/main/java/io/dropwizard/auth/PrincipalContainerRequestValueFactory.java

```diff
@@ -0,0 +1,23 @@
+package io.dropwizard.auth;
+
+import org.glassfish.jersey.server.internal.inject.AbstractContainerRequestValueFactory;
+
+import java.security.Principal;
+
+/**
+ * A value factory which extracts the {@link Principal} from the
+ * current {@link ContainerRequest} instance.
```

Comment:
Need to provide `import org.glassfish.jersey.server.ContainerRequest;` to use the short name.